sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -150,6 +151,30 @@ abstract class JdbcDialect extends Serializable {
   def beforeFetch(connection: Connection, properties: Map[String, String]): Unit = {
   }
 
+  /**
+   * Do some extra properties validation work in addition to the validation in [[JDBCOptions]],
+   * eg. [[validateFetchSize()]].
```

Comment:
Please update the doc:
```scala
[error] /home/jenkins/workspace/NewSparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/jdbc/JdbcDialect.java:121: error: reference not found
[error]    * Do some extra properties validation work in addition to the validation in {@link JDBCOptions},
[error]                                                                                       ^
[error] /home/jenkins/workspace/NewSparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/jdbc/JdbcDialect.java:122: error: reference not found
[error]    * eg. {@link validateFetchSize()}.
[error] 
```

https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/4903/consoleFull

Reply:
got it, thanks.